hera_qm/tests/test_xrfi.py

```diff
@@ -199,11 +262,13 @@ def rfi_gen():
                 yield data, rfi
             return
         self.rfi_gen = rfi_gen
-        self.ans['detrend_deriv'] = (.83, .1)
-        self.ans['detrend_medminfilt'] = (.2, .1)
-        self.ans['xrfi'] = (.75, .1)
-        self.ans['xrfi_simple'] = (.90, .1)
-
+        self.ans['detrend_deriv'] = [(.83, .83, .83),(.1,.1,.1)]
+        self.ans['detrend_medminfilt'] = [(.2, .3, .3),(.1,.1,.1)]
+        self.ans['xrfi'] = (.7, .1)
+        self.ans['xrfi_simple'] = [(.90, .90),(.1, .1)]
+        self.mode['detrend_deriv'] = [False,False,True]
+        self.mode['detrend_medminfilt'] = [False,False,False]
+        self.mode['watershed'] = [True,True]
 # class TestHERA(Template, unittest.TestCase):
```

Comment:
I know you didn't write these commented tests, but could you look them over and determine if we really want to keep them? If we want them, they should be run. If we don't want them, let's delete.
This is also the only place where `FILES` is used (see above)